incubator/knative/templates/eventing/clusterrole.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.eventing.install -}}
+{{ if .Values.rbac.enable }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
split the different resources into separate files (cluster role, role binding, serviceaccount etc)

Reply:
Also addressed